revup/topic_stack.py

```diff
@@ -179,6 +179,21 @@ class Topic:
     # Reviews for this topic, keyed by base branch
     reviews: Dict[str, Review] = field(default_factory=dict)
 
+    def depth_first_iter(self) -> Iterator[Topic]:
+        for t in self.depth_first_iter_helper(set()):
+            yield t
```

Comment:
This can just be `return self.depth_first_iter_helper(set())` right?  Or at least `yield from self.depth_first_iter_helper(set())`?

Reply:
til about yield from

Reply:
i think return is more appropriate here though